deploy/cr.yaml

```diff
@@ -178,6 +178,10 @@ spec:
       persistentVolumeClaim:
 #        storageClassName: standard
 #        accessModes: [ "ReadWriteOnce" ]
+#        dataSource:
+#          name: new-snapshot-test
+#          kind: VolumeSnapshot
+#          apiGroup: snapshot.storage.k8s.io
```

Comment:
@inelpandzic we didn't test this with a `VolumeSnapshot` right? I think we need to do it before merging this.

Reply:
@egegunes as expected, `VolumeSnapshot` also work. We can merge this.